roles/openshift_facts/library/openshift_facts.py

```diff
@@ -395,6 +395,23 @@ def set_metrics_facts_if_unset(facts):
             facts['common']['use_cluster_metrics'] = use_cluster_metrics
     return facts
 
+def set_openshift_metrics_facts_if_unset(facts):
+    """ Set openshift metrics facts if not already present in facts dict
+            dict: the facts dict updated with the generated openshift metrics facts if
+            missing
+        Args:
+            facts (dict): existing facts
+        Returns:
+            dict: the facts dict updated with the generated openshift metrics
+            facts if they were not already present
+
+    """
+    if 'common' in facts:
+        if 'use_cluster_metrics_hawkular' not in facts['common']:
+            use_cluster_metrics_hawkular = False
+            facts['common']['use_cluster_metrics_hawkular'] = use_cluster_metrics_hawkular
```

Comment:
@jdnieto Rather than creating a `set_openshift_metrics_facts_if_unset` function, @detiber is saying that we should move the fact setting to `get_defaults`.